### PR TITLE
boards: Set full_name for all boards

### PR DESCRIPTION
**Pull request by kartben**
Full name was set in all board.yml files corresponding board documentation, or in Twister files.
Whenever applicable, vendor name was dropped wherever it was prefixing the full name so that all boards have a consistent naming scheme.

I've tried to unify naming schemes across board series when I spotted some (ex. some STM32 boards had underscores in them when it should have been dashes), but I'm sure there's still a few left :)

Full list of the "new" full names, sorted alphabetically:

```
acn52832
ACRN hypervisor
ACRN on EHL hypervisor
AD-APARD32690-SL
ADP-XC7K AE350
Advanced BLE Cell
Aerocore2
Agilex SoC Development Kit
Agilex™ 5 SoC FPGA Development Kit
Alder Lake CRB
Alder Lake RVP
Apollo3 Blue EVB
Apollo3 Blue Plus EVB
Apollo4 Blue Plus KXR EVB
Apollo4P EVB
ARC EM Software Development Platform
ARC EM Starter Kit
ARC HS Development Kit
ARC HS4x/HS4xD Development Kit
ARC IoT Development Kit
ARC nSIM and HAPS FPGA boards
Arduino Due
Arduino GIGA R1 WiFi
Arduino MKR Zero
Arduino Nano 33 BLE (Sense)
Arduino Nano 33 IOT
Arduino Nicla Sense ME
Arduino Nicla Vision
Arduino OPTA
Arduino Portenta H7
Arduino UNO R4 Minima
Arduino UNO R4 WiFi
Arduino/Genuino Zero
Argon
Argonkey
ARMv8 Xen Virtual Machine Example
Arturo182 Serpente
Arty
AST1030_EVB
ATOM Lite
AtomS3
AtomS3 Lite
Avenger96
AZ3166 MXChip IoT DevKit
B-G474E-DPOW1 Discovery
B-L072Z-LRWAN1 Discovery kit
B-L4S5I-IOT01A Discovery kit
B-U585I-IOT02A Discovery kit
BASE RevC AEMv8A Fixed Virtual Platforms
BCM958401M2
BCM958402M2 (Cortex-M7)
BeagleConnect Freedom
BeaglePlay (CC1352)
BeagleV®-Fire
BL5340 DVK
BL652 DVK
BL653 DVK
BL654 DVK
BL654 Sensor Board
BL654 USB (451-00004)
Black Pill V1.2
Black Pill V2.0
Black Pill V3.0
Black STM32 F407VE Development Board
Black STM32 F407ZG Pro Development Board
BLE400
Blue Clover PLT Demo V2 nRF52832
Boron
byteSENSI-L
Carbon
CC1352P1 LaunchXL
CC1352P7 LaunchPad
CC1352R SensorTag
CC1352R1 LaunchXL
CC26x2R1 LaunchXL
CC3220SF LaunchXL
CC3235SF LaunchXL
Clicker 2 for STM32
Clicker RA4M1
Connectivity Card nRF52840
Core2
Cortex-M Trace Reference Board V1.2
Cortex-R8 Virtual
CX1825 nRF52840
CY8CPROTO-062-4343W
CY8CPROTO-063-BLE
Cyclone® V SoC Development Kit
CYW920829M2EVK-02
DA14695 Development Kit USB
DA1469x Development Kit Pro
Debug with Arm DS
Decawave DWM1001
Degu Evaluation Kit
Disco L475 IOT01 (B-L475E-IOT01A)
Dragonclaw Development Board
E73-TBB
Edge-V
EFM32 Giant Gecko 11 (SLSTK3701A)
EFM32 Happy Gecko (SLSTK3400A)
EFM32 Pearl Gecko (SLSTK3401A)
EFM32 Pearl Gecko 12 (SLSTK3402A)
EFM32 Wonder Gecko (EFM32WG-STK3800)
EFM32GG12 Thunderboard (SLTB009A)
EFR32BG13 2.4 GHz 10 dBm (SLWRB4104A)
EFR32BG22 Thunderboard (SLTB010A)
EFR32FG1 2400/868 MHz 13 dBm Dual Band (SLWRB4250B)
EFR32FG13 2400/915 MHz 19 dBm Dual Band (SLWRB4255A)
EFR32MG12 2.4 GHz 19 dBm (SLWRB4161A)
EFR32MG12 2400/868-915 MHz 19 dBm Dual Band (SLWRB4170A)
EFR32MG12 Thunderboard (SLTB004A)
EFR32xG21 2.4 GHz 20 dBm (SLWRB4180A)
EFR32xG24 2.4 GHz 20 dBm (xG24-RB4187C)
EFR32xG24 Dev Kit (xG24-DK2601B)
EFR32xG27 Dev Kit (xG27-DK2602A)
Elkhart Lake CRB
ENE KB1200_EVB
ESP-WROVER-KIT
ESP32-C3-DevKit-RUST
ESP32-C3-DevKitC
ESP32-C3-DevKitM
ESP32-C6-DevKitC
ESP32-DevKitC-WROOM
ESP32-DevKitC-WROVER
ESP32-Ethernet-Kit
ESP32-EVB
ESP32-S2 Franzininho
ESP32-S2 Lolin Mini
ESP32-S2-DevKitC
ESP32-S2-Saola
ESP32-S3-DevKitC
ESP32-S3-DevKitM
ESP32-S3-EYE
ESP32-S3-Touch-LCD-1.28
ESP32C3 0.42 OLED
ESP32C3_LUATOS_CORE
ESP32S3-Luatos-Core
ESP8684-DevKitM
EVAL-ADIN1110EVB Evaluation board
EVAL-ADIN2111EVB Evaluation board
EVK NINA-B11x
EVK NINA-B40x
EVK-ANNA-B11x
EVK-BMD-30/35: BMD-300-EVAL, BMD-301-EVAL, and BMD-350-EVAL
EVK-BMD-330: BMD-330-EVAL
EVK-BMD-34/38: BMD-340-EVAL and BMD-341-EVAL
EVK-BMD-34/38: BMD-345-EVAL
EVK-BMD-34/48: BMD-380-EVAL
EVK-BMD-360: BMD-360-EVAL
EVK-NINA-B3
Feather M0 Basic Proto
Feather M0 LoRa
Feather nRF52840 (Express, Sense)
Feather STM32F405 Express
FireCuda Gaming SSD (FaZe) board
FK7B0M1-VBT6
FMURT6
FRDM-K22F
FRDM-K64F
FRDM-K82F
FRDM-KE15Z
FRDM-KE17Z
FRDM-KE17Z512
FRDM-KL25Z
FRDM-KW41Z
FRDM-MCXA156
FRDM-MCXC242
FRDM-MCXC444
FRDM-MCXN236
FRDM-MCXN947
FRDM-MCXW71
FRDM_RW612
Ganymed Break-Out-Board (BOB)
GD32A503V-EVAL
GD32E103V-EVAL
GD32E507V-START
GD32E507Z-EVAL
GD32F350R-EVAL
GD32F403Z-EVAL
GD32F407V-START
GD32F450I-EVAL
GD32F450V-START
GD32F450Z-EVAL
GD32F470I-EVAL
GD32L233R-EVA
GD32VF103C-STARTER
GD32VF103V-EVAL
Generic LEON3
GR716-MINI Development Board
Grand Central M4 Express
Hexiwear
HiFive Unleashed
HiFive Unmatched
HiFive1
i.MX 7 Computer on Module - Colibri iMX7
i.MX 8QuadMax Multisensory Enablement Kit (MEK)
i.MX 8QuadXPlus Multisensory Enablement Kit (MEK)
i.MX 8ULP Evaluation Kit
i.MX8MM EVK
i.MX8MN EVK (Cortex-A53)
i.MX8MP EVK
i.MX93 EVK
i.MX95 EVK
Icarus
Icarus Bee
Icarus SoM
Icarus SoM DK
ICE-V Wireless
Integrated Sensor Hub (ISH) 5.4.1
Integrated Sensor Hub (ISH) 5.6.0
Integrated Sensor Hub (ISH) 5.8.0
Intel ADSP
INTEL FPGA niosv_g
INTEL FPGA niosv_m
IP Switch Board
IT82XX2 series
IT8XXX2 series
ItsyBitsy M4 Express
ItsyBitsy nRF52840
KB2040
KC868-A32
KV260 Development Board RPU Cortex-R5
Labs Blip
Labs Papyr
Legend
Linum Board
LiteX VexRiscv
LoDev
Longan Nano
LoRa STM32WL DevKit
LoRa-E5 Dev Board
LoRa-E5 mini
LPCXpresso11U68
LPCXPRESSO51U68
LPCXPRESSO54114
LPCXpresso55S06
LPCXpresso55S16
LPCXpresso55S28
LPCXpresso55S36
LPCXPRESSO55S69
LS1046A RDB
LSN50 LoRA Sensor Node
M2GL025 Mi-V
M5StickC PLUS
MAX10
MAX32655EVKIT
MAX32655FTHR
MAX32662EVKIT
MAX32666EVKIT
MAX32666FTHR
MAX32670EVKIT
MAX32672EVKIT
MAX32672FTHR
MAX32675EVKIT
MAX32680EVKIT
MAX32690EVKIT
MAX32690FTHR
MDBT50Q-DB-33
MDBT50Q-DB-40
MDBT53-DB-40
MDBT53V-DB-40
MEC1501 Modular card ASSY6885
MEC15xxEVB ASSY6853
MEC172x Modular Card ASSY6930 (Rev. B)
MEC172xEVB ASSY6906
Meerkat96
MERCURY-XU
micro:bit
micro:bit V2
MicroMod board Processor
MIMX8MQ EVK
MIMXRT1010-EVK
MIMXRT1015-EVK
MIMXRT1020-EVK
MIMXRT1024-EVK
MIMXRT1040-EVK
MIMXRT1050-EVK
MIMXRT1060-EVK
MIMXRT1060-EVKB
MIMXRT1064-EVK
MIMXRT1160-EVK
MIMXRT1170-EVK/EVKB
MIMXRT1180-EVK
MIMXRT595-EVK
MIMXRT685-EVK
MINI-M4 for STM32
MiniSTM32H743 Core Board
mpfs_icicle
MPS3 AN547
MR-CANHUBK3
MSP-EXP432P401R LaunchXL
MT8195 ADSP
Native POSIX execution (native_posix)
Native simulator - native_sim
NBSN95 NB-IoT Sensor Node
Neo Full
Neonkey
NEORV32
Nitrogen
NPCM400_EVB
NPCX4M8F_EVB
NPCX7M6FB_EVB
NPCX9M6F_EVB
nRF21540 DK
nRF51 DK
nRF51 Dongle
nRF51-VBLUno51
nRF52 Adafruit Feather
nRF52 DK
nRF52 simulated board (BabbleSim)
nRF52-VBLUno52
nRF52832 breakout
nRF52832-mdk
nRF52833 DK
nRF52840 DK
nRF52840 Dongle
nRF52840 MDK USB Dongle
nRF52840-mdk
nRF5340 Audio DK
nRF5340 DK
nRF5340 simulated boards (BabbleSim)
nRF54H20 DK
nRF54L15 DK
nRF54L15 PDK
nRF54L15 simulated boards (BabbleSim)
nRF54L20 PDK
nRF7002 DK
nRF9131 EK
nRF9151 DK
nRF9160 DK
nRF9160 Feather
nRF9160 INNBLUE21
nRF9160 INNBLUE22
nRF9160 Thing Plus
nRF9161 DK
nRF9280 PDK
Nucleo C031C6
Nucleo F030R8
Nucleo F031K6
Nucleo F042K6
Nucleo F070RB
Nucleo F091RC
Nucleo F103RB
Nucleo F207ZG
Nucleo F302R8
Nucleo F303K8
Nucleo F303RE
Nucleo F334R8
Nucleo F401RE
Nucleo F410RB
Nucleo F411RE
Nucleo F412ZG
Nucleo F413ZH
Nucleo F429ZI
Nucleo F446RE
Nucleo F446ZE
Nucleo F722ZE
Nucleo F746ZG
Nucleo F756ZG
Nucleo F767ZI
Nucleo G031K8
Nucleo G070RB
Nucleo G071RB
Nucleo G0B1RE
Nucleo G431RB
Nucleo G474RE
Nucleo H503RB
Nucleo H533RE
Nucleo H563ZI
Nucleo H723ZG
Nucleo H743ZI
Nucleo H745ZI-Q
Nucleo H753ZI
Nucleo H755ZI-Q
Nucleo H7A3ZI-Q
Nucleo L011K4
Nucleo L031K6
Nucleo L053R8
Nucleo L073RZ
Nucleo L152RE
Nucleo L412RB-P
Nucleo L432KC
Nucleo L433RC
Nucleo L452RE
Nucleo L476RG
Nucleo L496ZG
Nucleo L4A6ZG
Nucleo L4R5ZI
Nucleo L552ZE Q
Nucleo U031R8
Nucleo U083RC
Nucleo U575ZI Q
Nucleo U5A5ZJ Q
Nucleo WB05KZ
Nucleo WB09KE
Nucleo WB55RG
Nucleo WBA52CG
Nucleo WBA55CG
Nucleo WL55JC
NUMAKER M2L31KI
NUMAKER PFM M467
NUMAKER PFM M487
OBC module 1
ODROID-GO
OLIMEX-STM32-E407
OLIMEX-STM32-H103
OLIMEX-STM32-H405
OLIMEX-STM32-H407
OLIMEX-STM32-P405
OLIMEXINO-STM32
Open103Z
OpenISA VEGAboard
OpenTitan Earl Grey
Ophelia-I EV NRF52805
PAN1770 Evaluation Board
PAN1780 Evaluation Board
PAN1781 Evaluation Board
PAN1782 Evaluation Board
PAN1783 Evaluation Board
PAN1783A Evaluation Board
PAN1783A-PA Evaluation Board
PhyBOARD Polis (NXP i.MX8M Mini)
PhyBOARD Pollux (NXP i.MX8M Plus)
phyBOARD-Electra AM64x M4F Core
phyBOARD-Lyra AM62x A53 Core
Pico-Pi i.MX7D - Android Things IoT Development Platform
PineTime DevKit0
Pinnacle 100 DVK
Pro Micro RP2040
Proteus-II-EV
Proteus-III-EV
PSOC 62S4 Pioneer Kit
PSoC6 WiFi-BT Pioneer Kit
PSoC63 BLE Pioneer Kit
QEMU Emulation for Altera Nios-II
QEMU Emulation for ARCv2 & ARCv3
QEMU Emulation for ARM AArch64 Virt KVM
QEMU Emulation for ARM Cortex-A53
QEMU Emulation for ARM Cortex-M0
QEMU Emulation for ARM Cortex-M3
QEMU Emulation for ARM Cortex-R5
QEMU Emulation for Cortex-A9
QEMU Emulation for LEON3
QEMU Emulation for MIPS Malta
QEMU Emulation for RISCV32
QEMU Emulation for RISCV32 XIP
QEMU Emulation for RISCV32E Emulation
QEMU Emulation for RISCV64
QEMU Emulation for X86
QEMU Emulation for X86 / Lakemont CPU
QEMU Emulation for X86 64bit
QEMU Emulation for X86 Minimal Configuration
QEMU Emulation for Xtensa
Qomu
QT Py RP2040
QuickFeather
R-CAR H3 ARM CA57 (ARMv8)
R-Car H3 Salvator-X
R-CAR Salvator XS M3 ARM CA57 (ARMv8)
R-CAR Spider S4 (ARM64)
RA2A1 Evaluation Kit
RA4E2 Evaluation Kit
RA4M2 Evaluation Kit
RA4M3 Evaluation Kit
RA4W1 Evaluation Kit
RA6E1 Fast Prototyping Board
RA6E2 Evaluation Kit
RA6E2 Fast Prototyping Board
RA6M1 Evaluation Kit
RA6M2 Evaluation Kit
RA6M3 Evaluation Kit
RA6M4 Evaluation Kit
RA6M5 Evaluation Kit
RA8D1 Evaluation Kit
RA8M1 Evaluation Kit
RA8T1 Evaluation Kit
RAK11720
RAK4631
RAK5010
Raptor Lake P CRB
Raptor Lake S CRB
Raspberry Pi 4 Model B (Cortex-A72)
Raspberry Pi 5 (Cortex-A76)
Raspberry Pi Pico
RD-RW612-BGA
RDDRONE-FMUK66
RED-V Things Plus
Redbear Labs Nano
Redbear Labs Nano v2
reel board
RISC-V nSIM and HAPS FPGA boards
RISCV32 Virtual
RM1xx DVK
RoboKit 1
ROC-RK3568-PC (Quad-core Cortex-A55)
RuuviTag
SAM C21N Xplained Pro Evaluation Kit
SAM D20 Xplained Pro Evaluation Kit
SAM D21 Xplained Pro Evaluation Kit
SAM E54 Xplained Pro Evaluation Kit
SAM E70(B) Xplained
SAM L21 Xplained Pro Evaluation Kit
SAM R21 Xplained Pro Evaluation Kit
SAM R34 Xplained Pro Evaluation Kit
SAM V71(B) Xplained Ultra
SAM4E Xplained Pro
SAM4L-EK
SAM4S Xplained
SDP-K1
SECO SBC-3.5-PX30 (JUNO - D23) (STM32F302)
Seeeduino XIAO
SensorTile.box
SensorTile.box PRO
Sentrius BT510 Sensor
Sentrius BT610 Sensor
Sentrius™ MG100 Gateway
SiM3U1xx 32-bit MCU USB Development Kit
SK-AM62 M4F Core
ST25DV Discovery, MB1283 version
STAMP-C3
StampS3
Starter Kit+ for RZ/T2M
STEVAL STWINBX1 Development kit
STM32 Flight Controller Unit
STM32 M4 Clicker
STM32 Mini F401
STM32 Minimum Development Board
STM32 Sensor Mezzanine
STM3210C Evaluation
STM32373C Evaluation
STM32C0116-DK Discovery Kit
STM32F0 Discovery
STM32F030 DEMO BOARD
STM32F072 Evaluation
STM32F072B Discovery
STM32F103 Mini
STM32F3 Discovery
STM32F4 Discovery
STM32F405 Core Board V1.0
STM32F411E Discovery
STM32F412G Discovery
STM32F429I Discovery
STM32F469I Discovery
STM32F723E Discovery
STM32F746G Discovery
STM32F7508-DK Discovery Kit
STM32F769I Discovery
STM32G0316 Discovery
STM32G071B Discovery
STM32G081B Evaluation
STM32G431 Core Board
STM32H573I-DK Discovery
STM32H735G Discovery
STM32H745I Discovery
STM32H747I Discovery
STM32H750B Discovery Kit
STM32H7B3I Discovery kit
STM32H7S78-DK Discovery
STM32L1 Discovery
STM32L152C Discovery
STM32L475 Pandora
STM32L476G Discovery
STM32L496G Discovery
STM32L4R9I Discovery
STM32L562E-DK Discovery
STM32MP157C-DK2 Discovery
STM32U083C-DK
STM32U5A9J Discovery Kit
STM32VL Discovery
STM32WB5MM-DK
STM32WB5MMG
Swan
SwiftIO
SwiftIO Feather
Teensy 4.0
Teensy 4.1
THING PLUS MATTER
Thingy:52
Thingy:53
Titanium Ti60 F225
TLSR9518ADK80D
Trinket M0
TTGO LoRa32
TTGO T8-C3
Twinkie V2
TWR-KE18F
TWR-KV58F220M
UCANS32K1SIC
UP Squared
UP Squared Pro 7000
UPD301C Basic Sink Application Example
USB-KW24D512
V2M Beetle
V2M MPS2
V2M Musca B1
V2M Musca-S1
Verdin iMX8M Plus SoM
VisionFive 2 JH7110
VMU RT1170
W5500 Evaluation Pico
Walter
WaRP7 - Next Generation IoT and Wearable Development Platform
WGM160P Wi-Fi Module (SLWRB4321A)
WiFi LoRa 32 (V2)
Wio Terminal
Wireless Stick Lite (V3)
WisTrio
X-S32Z27X-DC (DC2)
Xenon
XIAO BLE (Sense)
XIAO ESP32C3
XIAO ESP32S3
XIAO RP2040
XMC45-RELAX-KIT
XMC47-RELAX-KIT
Xtensa simulator
YD-ESP32
YD-STM32H750VB
YJ-16019
Zybo

```

### CODE REVIEW

**Review by sylvioalves — APPROVED** [review]
LGTM for ESP32x
Edit: I suppose `ESP32C3_LUATOS_CORE` could be renamed to `ESP32S3-Luatos-Core` to match `ESP32S3-Luatos-Core`.

**Review by nashif — APPROVED** [review]

**Review by gmarull — CHANGES_REQUESTED** [review]
requires changes to board-schema.yml. May also be used by list_boards.py?

**Comment by kartben** [comment]
> requires changes to board-schema.yml. 

@gmarull 
https://github.com/zephyrproject-rtos/zephyr/pull/79425 :)

> May also be used by list_boards.py?

Ya I was thinking the same. I am assuming this would be non-blocking for this PR :)
Thanks!

**Review by gmarull — APPROVED** [review]
missed schema changes, lgtm

**Review by MaureenHelm — APPROVED** [review]

File: boards/qemu/riscv32e/board.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 board:
   name: qemu_riscv32e
+  full_name: QEMU Emulation for RISCV32E Emulation
```
Comment (tejlmand) [inline boards/qemu/riscv32e/board.yml:3]: why Emulation twice in this name ?

To be consistent with other Qemu boards:
```suggestion
  full_name: QEMU Emulation for RISCV32E
```

**Review by tejlmand — APPROVED** [review]
lgtm, a minor nit observed, but not blocking.

File: boards/qemu/riscv32e/board.yml (hunk 1 above)
Comment (kartben) [reply boards/qemu/riscv32e/board.yml:3]: definitely a typo! :) good catch - thank you

**Comment by kartben** [comment]
> Edit: I suppose `ESP32C3_LUATOS_CORE` could be renamed to `ESP32S3-Luatos-Core` to match `ESP32S3-Luatos-Core`.

@sylvioalves only seeing your edit now - it looks like there are a few other boards that are misusing underscores so I'll make another pass once this first batch is in.

File: boards/qemu/riscv32e/board.yml (hunk 1 above)
Comment (tejlmand) [reply boards/qemu/riscv32e/board.yml:3]: @kartben still present on main, will you do a followup PR ?
https://github.com/zephyrproject-rtos/zephyr/blob/6300e1bc253531b992c868424543cde112189915/boards/qemu/riscv32e/board.yml#L3

**Pull request merged**